src/SFA.DAS.EAS.Web.IntegrationTests/TestUtils/ApiTester/ApiIntegrationTester.cs

```diff
@@ -90,6 +91,8 @@ public Task<CallResponse> InvokeGetAsync(CallRequirements call)
 
         public Task<CallResponse<TResult>> InvokeGetAsync<TResult>(CallRequirements call)
         {
+            DbBuilder.EnsureTransaction();
```

Comment:
I'd argue that DbBuilder shouldn't have anything to do with invoking a call, it should only be used for setting up data prior to the call. Also interested to know why it's needed as maybe we missed something in the initial push to get these tests working again.

Reply:
I have implemented methods to get a new instance of the two DbBuilders, each with a DBContext  from a nested container of the API's container. DBBuilder is no longer used to set up the transaction.